src/app/plan/components/Wizard/types.ts

```diff
@@ -0,0 +1,35 @@
+// TODO add other properties used by components in this wizard, possibly move this somewhere more common
+// Maybe in the future we can somehow generate types like these from the CRDs in mig-controller?
+
+export interface IPlanPersistentVolume {
```

Comment:
+1 i am on board with the types.ts precedent for adding types to components going forward for now.